test/lib/redis-store-spec.js

```diff
@@ -84,6 +85,7 @@ describe('set', function () {
     });
   });
 
+  /*
   it('should store a null value without error', function (done) {
```

Comment:
We  shouldn't really leave commented the code and it is good to have null test (in case of the change to prove that null is not cached in the store) if there is not another one (can't really see from github PR)